cmd/erasure-metadata.go

```diff
@@ -34,7 +32,7 @@ import (
 	"github.com/minio/sha256-simd"
 )
 
-const erasureAlgorithmKlauspost = "klauspost/reedsolomon/vandermonde"
+const erasureAlgorithm = "reedsolomon"
```

Comment:
```suggestion
const erasureAlgorithm = "rs-vandermonde"
```
It is important we keep the matrix information in this. There may be better alternatives we could switch to in the future.

Reply:
Agreed